target/linux/rockchip/patches-5.4/105-rockchip-rk3328-add-support-for-FriendlyARM-NanoPi-N.patch

```diff
@@ -0,0 +1,75 @@
+From dbb35c0c4a4dddb40bf2b5dcc7a4208896ce6323 Mon Sep 17 00:00:00 2001
+From: Marty Jones <mj8263788@gmail.com>
+Date: Sat, 26 Dec 2020 05:08:15 -0500
+Subject: [PATCH] rockchip:rk3328: add support for FriendlyARM NanoPi NEO3
+
+This patch adds support for FriendlyARM NanoPi NEO3
```

Comment:
Please send this upstream.

Reply:
@blocktrron I sent the patch uptream [https://lore.kernel.org/linux-arm-kernel/20201228152836.02795e09.mj8263788@gmail.com/](url)

Reply:
linux-rockchip / linux-arm tends to ignore your patches if they are not sent to the maintainers themselves. So depending on whether this is still the case or not, you might need to resent the patch to a broader audience.